plugins/woocommerce-docs/src/Data/DocsStore.php

```diff
@@ -99,10 +108,9 @@ public static function update_docs_post( $post, $doc_id ) {
 			'tag'        => 'woocommerce_docs',
 		);
 
-		$existing_page = get_posts( $args );
+		$existing_doc = get_posts( $args );
 
-		if ( ! empty( $existing_page ) ) {
-			$post['ID'] = $existing_page[0]->ID;
```

Comment:
This isn't really needed, if you wanted to update the ID of the post for some reason you can pass it with the $post.